src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -296,72 +274,88 @@ class ContentSidebar extends PureComponent<Props, State> {
             return newView;
         }
 
-        return view;
+        // If the user has manually toggled respect that prior view,
+        // otherwise return the newly calculated view based on the above
+        // order of importance. This is mostly for making skills tab
+        // the priority always if skills are present.
+        return hasBeenManuallyToggled ? view : newView;
     }
 
     /**
-     * File fetch success callback that sets the file and view
-     * Only set file if there is data to show in the sidebar.
-     * Skills sidebar doesn't show when there is no data.
+     * Success callback for fetching metadata editors
      *
      * @private
      * @param {Object} file - Box file
      * @return {void}
      */
-    fetchMetadataSuccessCallback = (file: BoxItem, editors?: Array<MetadataEditor>): void => {
-        let newState = { isVisible: false };
-        if (SidebarUtils.shouldRenderSidebar(this.props, file, editors)) {
-            newState = {
-                file,
-                editors,
-                isVisible: true,
-                view: this.getDefaultSidebarView(this.props, file, editors),
-            };
-        }
-        this.setState(newState);
+    fetchMetadataSuccessCallback = ({ editors }: { editors: Array<MetadataEditor> }): void => {
+        const { file }: State = this.state;
+        this.setState({
+            metadataEditors: editors,
+            view: this.getDefaultSidebarView(file, editors),
+        });
     };
 
     /**
-     * File fetch success callback that sets the file and view
-     * Only set file if there is data to show in the sidebar.
-     * Skills sidebar doesn't show when there is no data.
+     * Fetches file metadata editors if required
      *
      * @private
-     * @param {Object} file - Box file
      * @return {void}
      */
-    fetchFileSuccessCallback = (file: BoxItem): void => {
+    fetchMetadata(): void {
```

Comment:
This function will remain and change, so will also account for skills data too, since skills is part of metadata. Right now we are fetching skills twice, one via /files?fields=metadata.global.boxSkillsCards and once via /files/metadata